File: Swift/8-forEach.swift
let a = [7, 10, 1, 5, 2]

a.forEach { x in
    print (x) // 7,10,1,5,2
}

a.forEach {
    print($0) // 7,10,1,5,2
}

[7,10,1,5].forEach {
    print($0) // 7,10,1,5
}

File: C++/8-forEach.cpp
#include "stdafx.h"
#include <iostream>
#include "conio.h"

using namespace std;

void ForEach()
{
	int arr[] = { 4, 2, 7, 8, 1, 6, 9, 5 };
	int index = 0;
	for each (int item in arr)
	{
		cout << index << ' ' << item << endl;
		index++;
	}

	_getch();
}